modular_skyrat/modules/colony_fabricator/code/machines/power_storage_unit.dm

```diff
@@ -0,0 +1,76 @@
+/obj/machinery/power/smes/battery_pack
+	name = "stationary battery"
+	desc = "An about table-height block of power storage, commonly seen in low storage high output power applications. \
```

Comment:
Same as before, since those descriptions are used for two items, might want to make it a define instead-

Actually, no, scratch that. Make it so that `/obj/item/flatpacked_machine/Initialize()` has a `desc = initial(type_to_deploy.desc)`, so we avoid code duplication everywhere here. I think it'd simplify adding more flatpacked machines in the future too.

Reply:
That's really good actually.